composeApp/src/commonMain/kotlin/sp/bvantur/tasky/home/ui/HomeScreen.kt

```diff
@@ -0,0 +1,22 @@
+package sp.bvantur.tasky.home.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+
+@Composable
+fun HomeRoute() {
+    HomeScreen()
+}
+
+@Composable
+fun HomeScreen() {
```

Comment:
Nitpick, if this Composable is only going to be called by `HomeRoute` then wouldn’t it make sense to mark this `HomeScreen` as `private`?